src/Libraries/CoreNodes/DSCoreNodes.Migrations.xml

```diff
@@ -26,7 +26,7 @@
   </priorNameHint>
   <priorNameHint>
     <oldName>DSCore.IO.CSV.WriteToFile</oldName>
-    <newName>DSCore.ImportExport.ExportCSV</newName>
+    <newName>DSCore.ImportExport.Import.ExportCSV</newName>
```

Comment:
I think this should be `Data.ExportCSV` as per the [spreadsheet](https://docs.google.com/spreadsheets/d/1IkmjLemPHfPhCfSW75R3NsUwlFkHpcVMDyvI8WvTiS0/edit#gid=0)